src/Noise.cpp

```diff
@@ -114,52 +129,82 @@ struct Noise : Module {
 		configOutput(BLACK_OUTPUT, "Black noise");
 		outputInfos[BLACK_OUTPUT]->description = "Uniform random numbers";
 
-		// Hard-code coefficients for Butterworth lowpass with cutoff 20 Hz @ 44.1kHz.
-		const float b[] = {0.00425611, 0.00425611};
-		const float a[] = {-0.99148778};
-		redFilter.setCoefficients(b, a);
+		SampleRateChangeEvent e;
```

Comment:
this part is no longer needed with latest Rack, which always calls `onSampleRateChange` for added modules.
see https://github.com/VCVRack/Rack/blob/v2/src/engine/Engine.cpp#L756